brianlong/www.validators.app
# 184205895 api for authorities changes

#### What's this PR do?
- api for account authority histories

#### How should this be manually tested?
- run tests
- run requests against `/api/v1/account-authorities/:network` and make sure everything works properly
- check API docs

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184205895)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
